server/models/Update/add_account_reference_module.sql

```diff
@@ -0,0 +1,12 @@
+/*
+  ACCOUNT REFERENCE MODULE AND REPORT
+  ===================================
+  NOTA : Please create `account_reference` and `account_reference_item` tables first
```

Comment:
@mbayopanda I think it would be better to include also the scripts of the creations of the tables that were added in the file schema.sql this will facilitate the update of the databases already (Tshikaji and Vanga)